backend/medtagger/api/scans/serializers.py

```diff
@@ -97,21 +99,11 @@
     'image_path': fields.String(),
```

Comment:
Does it make any sense to keep 'image_path' in ScanCategory? These images won't be shown anywhere, so for me they can be removed from the DB. What do you think?